installers/linux/al2/spec/java-amazon-corretto.spec.template

```diff
@@ -55,9 +55,18 @@
 %global abs2rel %{__perl} -e %{script}
 
 # Filter out private libraries so RPM does not list them as provided/required
-%global _privatelibs libjli[.]so.*
+%global _privatelibs libjli[.]so.*|libattach[.]so.*|libawt_headless[.]so.*|libawt[.]so.*|libdt_socket[.]so.*|libextnet[.]so.*|libfontmanager[.]so.*|libinstrument[.]so.*|libj2gss[.]so.*|libj2pcsc[.]so.*|libj2pkcs11[.]so.*|libjaas[.]so.*|libjavajpeg[.]so.*|libjdwp[.]so.*|libjimage[.]so.*|libjsound[.]so.*|libjsvml[.]so.*|liblcms[.]so.*|libmanagement[.]so.*|libmanagement_agent[.]so.*|libmanagement_ext[.]so.*|libmlib_image[.]so.*|libnet[.]so.*|libnio[.]so.*|libprefs[.]so.*|librmi[.]so.*|libsaproc[.]so.*|libsctp[.]so.*|libsyslookup[.]so.*|libzip[.]so.*|libsplashscreen[.]so.*|libawt_xawt[.]so.*
+%global _publiclibs libjawt[.]so.*|libjava.so.*|libjsig.so.*|libjvm.so.*|libverify.so.*
+
+%if "%{dist}" == ".amzn2" || "%{dist}" == ".amzn2int"
+# On AL2 we don't want to list any libraries as Provides to keep existing functionality
+%global __provides_exclude ^(%{_privatelibs}|%{_publiclibs})\$
+%global __requires_exclude ^(%{_privatelibs}|%{_publiclibs})\$
+%else
+# On newer AL we will want to latest LTS on release to provide the public libs.
```

Comment:
Nit - I think this comment has a missing word or grammatical error, not sure what it's intended to say

Reply:
Updating.